index.js

```diff
@@ -46,7 +46,7 @@ let atomicSDK = {
         inSdk: true,
         platform: {
           name: 'browser',
-          sdkVersion: '3.0.0',
+          sdkVersion: '3.0.2',
```

Comment:
@sean-hill When and why does this need to be updated?

Reply:
@wilsonparson it's an NPM package, so whenever you make a change and want to publish, we need to bump the package version, and then we run `npm publish --access public`. This looks great!